test_team_2.py

```diff
@@ -24,3 +24,21 @@ def test_uk_nhs(self):
 
     def test_it_passport(self):
         """Test it_passport functionality"""
+        # Positive test cas
+        prefix = ['CC', 'XX']
+        sufix = ['0000000', '1111111']
+        for p in prefix:
+            for s in sufix:
+                test_string = 'My passaportoy is ' + p + s
```

Comment:
All tests passed abd it looks goo, although I would like clarification on what passportoy the variable stands for. I'm a bit confused about the 'oy' part of the variable naming.